internal/cmd/cancel.go

```diff
@@ -0,0 +1,49 @@
+// Package cmd for handling commands
+package cmd
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/grid3-go/deployer"
+	"github.com/threefoldtech/grid3-go/internal/config"
+)
+
+// Cancel cancels a deployed project
+func Cancel(projectName string) error {
```

Comment:
lets add a cancel by project name function in th grid client itself so we can use it anywhare. what do you think?